ts/src/bybit.ts

```diff
@@ -7583,13 +7584,28 @@ export default class bybit extends Exchange {
         const request = {
             'mode': mode,
         };
-        if (symbol === undefined) {
-            request['coin'] = 'USDT';
-        } else {
-            const market = this.market (symbol);
+        const enableUnified = await this.isUnifiedEnabled ();
+        let method = undefined;
+        if (enableUnified[1]) {
+            this.checkRequiredSymbol ('setPositionMode', symbol);
+            market = this.market (symbol);
+            if (!market['inverse']) {
+                throw new NotSupported (this.id + ' setPositionMode () only support inverse market for unified trading account');
+            }
             request['symbol'] = market['id'];
+            request['category'] = 'inverse';
+            method = 'privatePostV5PositionSwitchMode';
+        } else {
+            if (symbol === undefined) {
+                request['coin'] = 'USDT';
+            } else {
+                market = this.market (symbol);
+                request['symbol'] = market['id'];
+            }
+            method = 'privatePostContractV3PrivatePositionSwitchMode';
```

Comment:
let's avoid dynamic calls plsss here we can easily do `this.privatePostContractV3PrivatePositionSwitchMode`